login-workflow/src/components/Error/ErrorManager.tsx

```diff
@@ -36,6 +36,7 @@ export type ErrorManagerProps = {
     dialogConfig?: {
         dismissLabel?: string;
         title?: string;
+        sx?: SxProps;
```

Comment:
This should be:
`sx?: SxProps<Theme>;`

Please update it in messageBoxConfig also.